repose-aggregator/components/services/datastore-service/datastore-service-api/build.gradle

```diff
@@ -12,7 +12,7 @@ jacocoTestCoverageVerification {
             rule.limits.forEach { limit ->
                 if (limit.counter == 'INSTRUCTION' && limit.value == 'COVEREDRATIO') {
                     // todo: set to the current coverage -- need more coverage, but we don't want the build to fail for the baseline
-                    limit.minimum = 0.07
+                    limit.minimum = 0.00
```

Comment:
is it all scala test code?

Reply:
Yep. I have no idea how it was passing previously.